app/views/sessions/login.html.erb

```diff
@@ -1,5 +1,6 @@
 <div>
   <h2>Login</h2>
+  <%= render 'shared/errors', obj: @user %>
   <%= form_tag '/login' do %>
     Username: <%= text_field_tag :username %></br>
     Password: <%= password_field_tag :password %></br>

```

Comment:
eto ung pang render ng errors sa log in page.
Obj: @user >> para ma-access niya ung errors.full_messages ni @user